modules/developer-cli-odo-switching-between-ephemeral-and-persistent-storage.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// *cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/working-with-storage.adoc
+
+[id="switching-between-ephemeral-and-persistent-storage_{context}"]
+= Switching between ephemeral and persistent storage
+
+You can switch between ephemeral and persistent storage in your project by using the `odo preference` command. `odo preference` modifies the global preference in your cluster.
+
+When persistent storage is enabled, the cluster stores the information between the restarts. 
+
+When ephemeral storage is enabled, the cluster stores temporary information that does not need to persist between the restarts.
+
+Persistent storage is enabled by default. 
```

Comment:
This changed with https://github.com/openshift/odo/pull/4360. Ephemeral is enabled by default. The user has to set it to false to use persistent storage.
```sh
$ odo preference -h | grep Ephemeral
 Ephemeral - If true odo will create a emptyDir volume to store source code (Default: true)
```